documentation/assertions/function/to-throw.md

```diff
@@ -24,19 +24,15 @@ You can assert the error message is a given string if you provide a
 string as the second parameter.
 
 ```js
-expect(
-  function() {
-    throw new Error('The error message');
-  },
-  'to throw',
-  'The error message'
-);
+expect(() => {
+  throw new Error('The error message');
+}, 'to throw', 'The error message');
```

Comment:
Personally, I find arrow-funcs spanning several lines can be quite difficult to read. I generally tend towards:

```js
expect(
  () => { throw new Error("The error message"); },
  "to throw", "The error message"
);
```

Reply:
We should aim at formatting te documentation with prettier. So whatever it decides.

Reply:
It says:

```js
expect(
  () => {
    throw new Error('The error message');
  },
  'to throw',
  'The error message'
);
```